CODEOWNERS

```
@@ -89,6 +90,7 @@
 /tests/lib/bm_zms/                        @nrfconnect/ncs-bm @rghaddab
 /tests/lib/ble_qwr/                       @nrfconnect/ncs-bm
 /tests/lib/ble_racp/                      @nrfconnect/ncs-bm
+tests/lib/ble_radio_notif/                @nrfconnect/ncs-bm
```

```suggestion
/tests/lib/ble_radio_notif/               @nrfconnect/ncs-bm
```
Seems like the other ones start with /

---

yes, thanks